x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_log_expanded_tray.tsx

```diff
@@ -292,7 +314,7 @@ export const ActionsLogExpandedTray = memo<{
         description: (
           // codeblock for output
           <StyledEuiCodeBlock data-test-subj={getTestId('details-tray-output')}>
-            <OutputContent action={action} data-test-subj={dataTestSubj} />
+            <OutputContent action={action} data-test-subj={dataTestSubj} fromAlertWorkaround />
```

Comment:
This should be `fromAlertWorkaround={fromAlertWorkaround}` to pass the value of the prop to `OutputContent`

Reply:
🤦‍♂️